src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/ExportModule.cs

```diff
@@ -83,7 +82,14 @@ private ContractDescription LoadContract(Type type)
 
             private static bool IsContractType(Type type)
             {
-                return (type.IsInterface || type.IsClass) && (type.IsDefined(typeof(ServiceContractAttribute), false));
+                Type serviceContractAttributeType = Tool.SMAssembly.GetType("System.ServiceModel.ServiceContractAttribute");
+                if(serviceContractAttributeType == null)
```

Comment:
This method is called repeatedly. serviceContractAttributeType should be cached.